samples/snippets/src/main/java/com/example/dialogflow/KnowledgeBaseManagement.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Google LLC
+ * Copyright 2022 Google LLC
```

Comment:
For handwritten code, the license year should be the year the code was first created

Reply:
Thanks! Updated.